tools/GeneratePythonLibrary.sh

```diff
@@ -90,7 +90,7 @@ function reset {
 
   echo "yapf3 -ir src/"
   if [ -x "$(command -v yapf3)" ]; then
-    yapf3 -ir src/
+    yapf3 -ir --style='{based_on_style: yapf, indent_width: 4}' src/
```

Comment:
tbh i kind of like google's 2-space indentation since it's more compact, but no strong feelings, open to whatever the group prefers.

https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#s3.4-indentation

Reply:
I sorta prefer two as well, and that's what's used in `kapigen`. It's easier to see the non-indent changes this way. If others agree then I think we can change to two before I merge this.

Reply:
Agreed on preferring compact.  Can you change it and then update PR #635 ?